platform/ios/app/MBXViewController.h

```diff
@@ -1,5 +1,14 @@
 #import <UIKit/UIKit.h>
+@class MBXState;
+@class MGLMapView;
 
 @interface MBXViewController : UIViewController
 
+@property (nonatomic) MBXState *currentState;
+@property (nonatomic, readonly) IBOutlet MGLMapView *mapView;
```

Comment:
Once you move the manager to receive state objects you don't have to expose this attributes.